packages/fury-adapter-apib-serializer/template.nunjucks

```diff
@@ -77,9 +78,9 @@ FORMAT: 1A
 
 {% macro renderTransition(transition, href) %}
 {% if transition.title and transition.title.toValue() %}
-#### {{ transition.title.toValue() }} [{{ transition.method.toValue() }}{% if href %} {{ href }}{% elif transition.computedHref and transition.computedHref.toValue() %} {{ transition.computedHref.toValue() }}{% endif %}]
+#### {{ transition.title.toValue() }} [{% if transition.method %}{{ transition.method.toValue() }}{% endif %}{% if href %} {{ href }}{% elif transition.computedHref and transition.computedHref.toValue() %} {{ transition.computedHref.toValue() }}{% endif %}]
```

Comment:
When an action is missing a response,

```apib
FORMAT: 1A

# API name

## Group Users

### User [/users]

#### List all users [GET]

Get the list of all the users
```

The output tree is

```json
{
  "element": "parseResult",
  "content": [
    {
      "element": "category",
      "meta": {
        "classes": {
          "element": "array",
          "content": [
            {
              "element": "string",
              "content": "api"
            }
          ]
        },
        "title": {
          "element": "string",
          "content": "API name"
        }
      },
      "attributes": {
        "metadata": {
          "element": "array",
          "content": [
            {
              "element": "member",
              "meta": {
                "classes": {
                  "element": "array",
                  "content": [
                    {
                      "element": "string",
                      "content": "user"
                    }
                  ]
                }
              },
              "content": {
                "key": {
                  "element": "string",
                  "content": "FORMAT"
                },
                "value": {
                  "element": "string",
                  "content": "1A"
                }
              }
            }
          ]
        }
      },
      "content": [
        {
          "element": "category",
          "meta": {
            "classes": {
              "element": "array",
              "content": [
                {
                  "element": "string",
                  "content": "resourceGroup"
                }
              ]
            },
            "title": {
              "element": "string",
              "content": "Users"
            }
          },
          "content": [
            {
              "element": "resource",
              "meta": {
                "title": {
                  "element": "string",
                  "content": "User"
                }
              },
              "attributes": {
                "href": {
                  "element": "string",
                  "content": "/users"
                }
              },
              "content": [
                {
                  "element": "transition",
                  "meta": {
                    "title": {
                      "element": "string",
                      "content": "List all users"
                    }
                  },
                  "content": [
                    {
                      "element": "copy",
                      "content": "Get the list of all the users"
                    }
                  ]
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}

```

And when you serialize this, the current master crashes.